testsuite/integration/web/src/test/java/org/jboss/as/test/integration/web/security/jaspi/WebSecurityJaspiTestCase.java

```diff
@@ -63,30 +66,39 @@
 public class WebSecurityJaspiTestCase extends WebSecurityPasswordBasedBase {
     private static final Logger log = Logger.getLogger(WebSecurityJaspiTestCase.class);
 
+    private static final String JBOSS_WEB_CONTENT = "<?xml version=\"1.0\"?>\n" +
+            "<jboss-web>\n" +
+            "    <security-domain>" + WebJaspiTestsSecurityDomainSetup.WEB_SECURITY_DOMAIN + "</security-domain>\n" +
+            "</jboss-web>";
+
     @Deployment
     public static WebArchive deployment() throws Exception {
         WebArchive war = ShrinkWrap.create(WebArchive.class, "web-secure-jaspi.war");
         war.addClass(SecuredServlet.class);
 
-        war.addAsWebInfResource(WebSecurityBASICTestCase.class.getPackage(), "jboss-web.xml", "jboss-web.xml");
+        war.addAsWebInfResource(new StringAsset(JBOSS_WEB_CONTENT), "jboss-web.xml");
         war.addAsWebInfResource(WebSecurityBASICTestCase.class.getPackage(), "web.xml", "web.xml");
 
-        war.addAsResource(WebSecurityBASICTestCase.class.getPackage(), "users.properties", "users.properties");
-        war.addAsResource(WebSecurityBASICTestCase.class.getPackage(), "roles.properties", "roles.properties");
-
         return war;
     }
 
     @ArquillianResource
     private URL url;
 
+    @BeforeClass
+    public static void beforeClass() {
+        // Until this Jira https://issues.jboss.org/browse/EAP7-649 is solved, we should not enable Elytron for this
+        // test case
+        AssumeTestGroupUtil.assumeElytronProfileTestsEnabled();
```

Comment:
Are the tests ignored because they fail when Elytron profile is used? The plan was to get rid of `AssumeTestGroupUtil.assumeElytronProfileTestsEnabled()` before EAP 7.1. If the tests just works (using legacy-based security) with Elytron profile enabled, let's don't skip them.